beoble/beoble-toolkit
# BEOBLE-1476-Fix-Vercel-staging-env-based-plugin-key-mixup

- add `styledComponents: true` to next.config.js in example project
- prevent html hydration error (nested p in p element)
- solving unmount component issue
- migration json
- fix hydration errors cleanup Button, Avatar, and Icon etc. components
- build fix
- delete log
- delete un-used svgs
- code review: make param optional
- rework web3 atom with wagmi
- injected ethereum check on browser
- remove unnecessary nullish coalesced values
- use usesign message instead of signer
- test logging infura id
- test with prefix next_public
- retest with next public prefix
- consisten icon names
- remove loading panel on connect page
- codify colors use colors in Button components set default values
- Updated Stickers and media links
- dynamic lazy imported svg icons with Suspense
- cleanup logic from const file
- fix breakpoint
- use theme colors
- remove useRouteAndRouterMeta hook
- cleanup useBottomFooterNav
- Disable logging in state for packages/beoble/components/Popup/index.tsx after login success https://kryptonium.atlassian.net/browse/BEOBLE-1420
- beoble vs logo icon in connecting popup
- implement unread chat badges with reusable AbsoluteCornerBadge component
- reduce NFTImage width and height to fit into min 390px width https://www.figma.com/file/1CPqj0Dqv2AtH7Ffjayj8h?node-id=3711:66909#414713755
- update badge for NFT with AbsoluteCornerBadge
- Show unread chat count in bottom footer regardless of current route https://kryptonium.atlassian.net/browse/BEOBLE-1436
- Review updates and the search bar
- Fixed edit profile and chatinput icons
- James review updates
- Fixed Nft displayer sizes
- remove window.ethereum condition
- logging with alert
- pass sign message async
- logging injected account
- adding login fail log
- provider only when there is a injected etherum on window
- using only provider from injected
- clean up
- fixing falsy jsx element 'PopUp'
- Created outside clicking hook and  updated reviewed stuff
- remove log and merge dev
- using forbidden ts-ignore to build temporaly. this should be handled more properly later
- applying comments
- global loading false by default
- cleanup
- cleanup
- [hotfix] for sign message in first user registration
- Updated stream and lastmessage into useChannel
- Updated Lastmessage hook
- reduce requests to backend - mostly done by getting individual chatroom from total chatrooms list but in general many other improvements need to be made (must look more into staleTime etc. - right now the way I have been configuring staleData is causing data to not update properly)
- fix hashloader position
- prevent online status for self
- Updated background height
- Update index.tsx
- Reducing height (tested on simulator)
- Fixed unread chat indicator
- cleanup
- react query client config staleTime 5s to prevent excessive requests
- use react query mutations for profile edit
- repsonsive sizes for protfolio tokens
- [hotfix] use user init
- change to chatroom read
- hotfix-revert-staleTime-react-query-data-config
- get user when arriving at root route
- test commit for debugging
- revert test changes
- revert changes
- [hotfix] mark as read
- Fixed username bug and system message
- fixed adding members to the group
- Fixed chatroom scroll issue
- removed screen.orintation
- size fix in chatroom create
- correct ismember changed logic
- Header avatar fix
- change to use global font family
- remove unusing lines
- log env values
- use `development` and `preview` for values to check against process.env.VERCEL_ENV (instead of process.env.NEXT_PUBLIC_VERCEL_ENV) due to _document.tsx being server-side rendered
- test - swap production and preview deliberately
- revert test change
- test 2
- revert test 2
- delete logs
- restore script order
- test 3
- revert test
